covalent/_results_manager/result.py

```diff
@@ -372,6 +375,27 @@ def _get_node_output(self, node_id: int) -> Any:
         """
         return self._lattice.transport_graph.get_node_value(node_id, "output")
 
+    def _get_node_qelectron_data(self, node_id: int) -> dict:
+        """
+        Return all QElectron data associated with a node.
+
+        Args:
+            node_id: The node id.
+
+        Returns:
+            The QElectron data of said node. Will return None if no data exists.
+        """
+        try:
+            # Checks existence of QElectron data.
+            self._lattice.transport_graph.get_node_value(node_id, "qelectron_data_exists")
+        except KeyError:
+            return None
+
+        results_dir = get_config("dispatcher")["results_dir"]
+        db_dir = os.path.join(results_dir, self.dispatch_id, QE_DB_DIRNAME)
```

Comment:
@kessler-frost does this look directly into the server's filesystem? Will it work with a self-hosted deployment?

Reply:
It does access the server's filesystem directly. Which means this won't work in self-hosted deployment or in the case of cloud executors.